pages/_app.tsx

```diff
@@ -132,6 +141,7 @@ export default function App({ Component, pageProps }: AppProps) {
               </dialog>
             </ModalContext.Provider>
             <Toast />
+            {isPaymentDialogOpen && <RatioWrapper />}
```

Comment:
see previous comment above re: conditional rendering, I can foresee us doing a `translateX` in the future from the previous screen.